images/protected-publish/README.md

```diff
@@ -189,10 +231,13 @@ To examine a local or remote (S3 only) index for any missing specs:
 docker run --rm \
     --entrypoint python \
     -ti protected-publish:latest \
-    validate_index.py --url s3://spack-binaries/develop-2024-01-07
+    -m pkg.validate_index \
+    --url s3://spack-binaries/develop-2024-01-07 --version 2
 ```
 
-Optionally you can specify a local file instead with `--file` and an absolute path.
+The version is used to select whether v2 or v3 index is sought.
+
+Optionally, instead of specifying `--url` and `--version`, you can specify a local file with `--file` followed by an absolute path.
```

Comment:
is this `--file` argument what `develop_stacks.txt` and `test_stacks.txt` is meant to be used for?

Reply:
No, looks like this is is the `validate` functionality, so `--file` is supposed to be a buildcache index.  I'll update the text.

Those files are meant to be pasted in to the `migrate_job.yaml` before running it to migrate a particular swath of mirrors.  Maybe they don't belong in the repo?